packages/public/camera-web/src/Camera/hooks/useUserMedia.ts

```diff
@@ -133,6 +133,24 @@ function getStreamDimensions(stream: MediaStream): PixelDimensions {
   return { width, height };
 }
 
+function swapWidthAndHeight(dimensions: PixelDimensions): PixelDimensions {
+  return {
+    width: dimensions.height,
+    height: dimensions.width,
+  };
+}
+
+function isMobileDevice(): boolean {
```

Comment:
Add this function in the common package, in a util file called `browser.utils.ts` and don't forget to add tests